packages/escape-html/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.0.2 (2019-??-??)
```

Comment:
Is this the right format? I saw previous additions with `(Unreleased)`, unsure if this is matched anywhere. Cc @gziolo.

Reply:
`(Unreleased)` is the preferred one. At least it is documented this way:

https://github.com/WordPress/gutenberg/tree/master/packages#maintaining-changelogs